Mathlib/Topology/CWComplex/Basic.lean

```diff
@@ -0,0 +1,682 @@
+/-
+Copyright (c) 2024 Floris van Doorn and Hannah Scholz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn, Hannah Scholz
+-/
+
+import Mathlib.Analysis.InnerProductSpace.Basic
+import Mathlib.Data.Real.StarOrdered
+import Mathlib.Order.CompletePartialOrder
+
+/-!
+# CW-complexes
+
+This file defines (relative) CW-complexes and proofs basic properties about them.
+
+A CW-complex is a topological space that is made by gluing closed disks of different dimensions
+together.
+
+## Main definitions
+* `RelCWComplex C D` : The class of CW-structures on a subspace `C` realtive to `D` of a topological
+  space `X`.
+* `ClasCWComplex C` : An abbreviation for `RelCWComplex C ∅`. The class of CW-structures on a
```

Comment:
I'm surprised by the abbreviation `ClasCWComplex`: does this stand for "classical"? For the class of all CW complexes? (Then I'd expect at least `ClassCWComplex`.)

Reply:
Yes, I did this spontaneously to get CI to stop complaining about the name being the same as for the other definition. It should definitely be different. I meant it to stand for classical.